### PR TITLE
Add `getCommitteeAddress`

### PR DESCRIPTION
**Pull request by shargon**
- Reviewed null types in `TestEngine`.
- Added `getCommitteeAddress`.

### CODE REVIEW

File: src/Neo.SmartContract.Testing/Native/OracleContract.cs [hunk 1]
```diff
@@ -13,12 +12,12 @@ public abstract class OracleContract : SmartContract, TestingStandards.IVerifica
 
     #region Events
 
-    public delegate void delOracleRequest(ulong? Id, UInt160? RequestContract, string? Url, string? Filter);
+    public delegate void delOracleRequest(ulong Id, UInt160 RequestContract, string Url, string Filter);
```
Comment (shargon) [inline src/Neo.SmartContract.Testing/Native/OracleContract.cs:15]:
```suggestion
    public delegate void delOracleRequest(ulong Id, UInt160 RequestContract, string Url, string? Filter);
```

File: src/Neo.SmartContract.Framework/Native/NEO.cs [hunk 2]
```diff
@@ -39,6 +39,8 @@ public class NEO
         public static extern Iterator<(ECPoint, BigInteger)> GetAllCandidates();
         public static extern BigInteger GetCandidateVote(ECPoint pubkey);
         public static extern ECPoint[] GetCommittee();
+
```
Comment (shargon) [inline src/Neo.SmartContract.Framework/Native/NEO.cs:42]:
```suggestion
```

File: src/Neo.SmartContract.Testing/Native/NeoToken.cs [hunk 3]
```diff
@@ -5,31 +5,30 @@
...
...urntype"":""Boolean"",""offset"":112,""safe"":false},{""name"":""unclaimedGas"",""parameters"":[{""name"":""account"",""type"":""Hash160""},{""name"":""end"",""type"":""Integer""}],""returntype"":""Integer"",""offset"":119,""safe"":true},{""name"":""unregisterCandidate"",""parameters"":[{""name"":""pubkey"",""type"":""PublicKey""}],""returntype"":""Boolean"",""offset"":126,""safe"":false},{""name"":""vote"",""parameters"":[{""name"":""account"",""type"":""Hash160""},{""name"":""voteTo"",""type"":""PublicKey""}],""returntype"":""Boolean"",""offset"":133,""safe"":false}],""events"":[{""name"":""CandidateStateChanged"",""parameters"":[{""name"":""pubkey"",""type"":""PublicKey""},{""name"":""registered"",""type"":""Boolean""},{""name"":""votes"",""type"":""Integer""}]},{""name"":""Vote"",""parameters"":[{""name"":""account"",""type"":""Hash160""},{""name"":""from"",""type"":""PublicKey""},{""name"":""to"",""type"":""PublicKey""},{""name"":""amount"",""type"":""Integer""}]},{""name"":""CommitteeChanged"",""parameters"":[{""name"":""old"",""type"":""Array""},{""name"":""new"",""type"":""Array""}]}]},""permissions"":[{""contract"":""*"",""methods"":""*""}],""trusts"":[],""extra"":null}");
```
Comment (Jim8y) [inline src/Neo.SmartContract.Testing/Native/NeoToken.cs:12]: oh man, this is not right, please do figure out a way to do it automatically.

**Review by Jim8y — APPROVED** [review]

File: src/Neo.SmartContract.Testing/Native/NeoToken.cs (hunk 3 above)
Comment (shargon) [reply src/Neo.SmartContract.Testing/Native/NeoToken.cs:12]: I generate it automatically

**Pull request merged**